test/appium/tests/test_transaction.py

```diff
@@ -151,6 +151,21 @@ def test_send_eth_from_wallet_sign_later(self):
         transactions_view.history_tab.click()
         transactions_view.transactions_table.find_transaction(amount=amount)
 
+    @pytest.mark.transactions
+    def test_open_transaction_on_etherscan(self):
+        user = transaction_users['A_USER']
+        sign_in_view = SignInView(self.driver)
+        sign_in_view.recover_access(user['passphrase'], user['password'])
+        home_view = sign_in_view.get_home_view()
+        wallet_view = home_view.wallet_button.click()
+        transactions_view = wallet_view.transactions_button.click()
+        transaction_details = transactions_view.transactions_table.get_first_transaction().click()
+        time.sleep(3) # Wait for transaction details screen to appear
+        transaction_details.options_button.click()
+        transaction_details.open_transaction_on_etherscan_button.click()
+        time.sleep(10) # Wait for external browser to appear
+        assert is_web_view_opened(self.driver)
```

Comment:
would be better to verify that there is unique amount of the transaction on the screen